app/views/vocabs/create.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.id @article.id
```

Comment:
modify `@article` for `@vocab` or whichever variable you send from the controller